packages/website/pages/account/payment.js

```diff
@@ -136,6 +136,15 @@ const PaymentSettingsPage = props => {
             <h1 className="table-heading">Payment</h1>
           </div>
           <div className="billing-content">
+            {currentPlan?.id === 'free' && !savedPaymentMethod && typeof paymentSettings === 'undefined' && (
+              <div className="add-billing-cta">
+                <p>
+                  You don&apos;t have a paid plan. Please add a credit/debit card and select a plan to prevent storage
```

Comment:
I think the `&& typeof paymentSettings === 'undefined'` part does anything? `currentPlan.id` should never be 'free' when `typeof paymentSettings === 'undefined'` because `currentPlan` is derrived from `paymentSettings` (see line 113).

If that analysis is right though, then I think the harm is low though, because I think that last part of the `&&` will never matter...

Reply:
It looked like that was being used for "fetching user info", but certainly I meant to evaluate to !== because I don't want this to show while the user is being fetched. In both cases it wouldn't matter because id isn't free you are correct.